ts/tests/testSongParser.ts

```diff
@@ -0,0 +1,78 @@
+// We'll probably end up moving some of these functions to the parser since we're storing a new JSON format
+import {reverseString, renderChord, renderChordFingering, renderChordLyricLine, renderLines, instrumentsData} from "../songParser";
+import { assert, expect } from "chai";
```

Comment:
Didn’t we decide to stick with `assert`? (`expect(…).to.eql(…)` → `assert.deepEqual(…, …)`.)